server/httpd/http.go

```diff
@@ -2,24 +2,55 @@ package httpd
 
 import (
 	"net/http"
+	log "github.com/sirupsen/logrus"
+	"path/filepath"
 )
 
-func NewSPA(path string, fs http.FileSystem) *SPA {
+func NewSPA(raw_index string, fs http.FileSystem) *SPA {
 	return &SPA{
-		path: path,
+		RawIndex: raw_index,
 		fs:   fs,
 	}
 }
 
 type SPA struct {
-	path string
+	RawIndex string
 	fs   http.FileSystem
 }
 
-func (s *SPA) Open(name string) (http.File, error) {
-	if file, err := s.fs.Open(name); err != nil {
-		return s.fs.Open(s.path)
+func (s* SPA) _index_serve(w http.ResponseWriter) {
+	w.Header().Add("Content-type", "text/html")
```

Comment:
```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

Reply:
When we can't found the requested filename. will response index.html.
Old code has the same thing.